src/Auth0.ManagementApi/Models/CreateGuardianEnrollmentTicketRequest.cs

```diff
@@ -5,20 +5,20 @@ namespace Auth0.ManagementApi.Models
     public class CreateGuardianEnrollmentTicketRequest
     {
         /// <summary>
-        ///     Alternate email to which the enrollment email will be sent. Optional - by default, the email will be sent to the
-        ///     user's default address
+        /// Alternate email to which the enrollment email will be sent. Optional - by default, the email will be sent to the
+        /// user's default address
         /// </summary>
         [JsonProperty("email")]
         public string EmailAddress { get; set; }
 
         /// <summary>
-        ///     Indicates whether an email must be sent to the user to start the enrollment.
+        /// Indicates whether an email must be sent to the user to start the enrollment.
         /// </summary>
         [JsonProperty("send_mail")]
-        public bool MustSendMail { get; set; }
+        public bool? MustSendMail { get; set; }
```

Comment:
this change is making it null safe? 🤔

Reply:
It's actually the opposite.  This can now be null, true or false.  This is required whenever we have an API that has optional fields on the REST API otherwise there is no way of not sending the value.

Reply:
Oh, and making something nullable is non-breaking because of the way the compiler/language works :)